doc/api/http2.md

```diff
@@ -283,6 +283,16 @@ session.setTimeout(2000);
 session.on('timeout', () => { /** .. **/ });
 ```
 
+#### http2session.alpnProtocol
+<!-- YAML
+added: REPLACEME
+-->
+
+Value will be `undefined` if the `Http2Session` is not yet connected to a
+socket, `h2c` if the `Http2Session` is not connected to a `TLSSocket`, or
```

Comment:
why `'h2c'`?

Reply:
`h2c` === HTTP/2 over plain text. I'm using it generically because, typically, when using a plain text connection, there's no `alpnProtocol` property we can use to source it. So if the socket is not a TLSSocket, we assume it's plain text.